android/java/org/chromium/chrome/browser/crypto_wallet/modal/BraveWalletPanel.java

```diff
@@ -119,41 +120,53 @@ public void onDismiss() {
         setUpViews();
     }
 
-    public void showPopupMenu() {
-        PopupMenu menu = new PopupMenu(mOptionsImage.getContext(), (View) mOptionsImage);
-        menu.getMenuInflater().inflate(R.menu.menu_dapps_panel, menu.getMenu());
-        menu.setOnMenuItemClickListener(item -> {
-            if (item.getItemId() == R.id.action_lock_wallet) {
-                mBraveWalletPanelServices.getKeyringService().lock();
+    private void showPopupMenu() {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.N_MR1) {
```

Comment:
nit: would be great to add a comment here about the crash and what exactly fixes it, as it seems like a typical android edge case (bug in the platform lib).

Reply:
it's a good point, let me add it.